packages/cosmwasm/src/cosmwasmclient.ts

```diff
@@ -103,24 +102,6 @@ export interface ContractDetails extends Contract {
   readonly initMsg: object;
 }
 
-/** A transaction that is indexed as part of the transaction history */
```

Comment:
This is all we remove here?

I was thinking that we get most of this for free by importing `CosmosClient from "@cosmwasm/sdk38"`.

It is not essential if this is going to be deprecated, but if this is a model for how we will do composition, adding more modules, it is nice to show the reuse.

Reply:
I'll deduplicate that code here, no problem